samples/counter_entity/RetrieveEntity/function.json

```diff
@@ -0,0 +1,25 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "authLevel": "function",
+      "name": "req",
+      "type": "httpTrigger",
+      "direction": "in",
+      "route": "entity/{entityName:alpha}/{entityKey}",
```

Comment:
Why do we need the :alpha in the route?

Reply:
Good point, we don't actually *need* it. I'll remove it to avoid any confusion.